src/platform/intel/cavs/dai.c

```diff
@@ -101,11 +103,13 @@ static struct dai dmic[2] = {
 static struct dai hda[(DAI_NUM_HDA_OUT + DAI_NUM_HDA_IN)];
 
 static struct dai_type_info dti[] = {
+#if defined CONFIG_CAVS_SSP
```

Comment:
`#if CONFIG__*` is usually better than `#ifdef CONFIG_*`, because when CONFIG_ will be 0/false it will still work

Reply:
Sure, fixed. I will also change that for DMIC in another PR.